planex/cmd/clone.py

```diff
@@ -70,13 +70,13 @@ def repo_name(url):
 """)
 
 
-def clone_jenkins(url, destination, commitish, credentials):
-    """Print Jenkinsfile fragment to clone repository"""
-    destination = join(destination, repo_name(url))
-    print(CHECKOUT_TEMPLATE.substitute(url=url,
```

Comment:
You can delete the definition of `CHECKOUT_TEMPLATE` above and also remove the `--credentials` argument.